File: demo/KSYLiveDemo/KSYDBCreater.m
//
//  KSYDBCreater.m
//  KSYPlayerDemo
//
//  Created by ksy on 16/4/25.
//  Copyright © 2016年 kingsoft. All rights reserved.
//

#import "KSYDBCreater.h"
#import "KSYSQLite.h"


@implementation KSYDBCreater

+ (void)initDatabase{
    NSString *key=@"IsCreatedDb";
    NSUserDefaults *defaults=[[NSUserDefaults alloc]init];
    if ([[defaults valueForKey:key] intValue]!=1) {
        [self createUrlTable];
        [defaults setValue:@1 forKey:key];
    }
}

+ (void)createUrlTable{
    NSString *sql=@"CREATE TABLE Address (address text)";
    [[KSYSQLite sharedInstance]executeNonQuery:sql];
}
@end

File: demo/KSYLiveDemo_Swift/KSYLiveDemo_Swift/KSYDBCreater.swift
//
//  KSYDBCreater.swift
//  KSYLiveDemo_Swift
//
//  Created by iVermisseDich on 17/1/13.
//  Copyright © 2017年 com.ksyun. All rights reserved.
//

import UIKit

class KSYDBCreater: NSObject {

    override init() {
        super.init()
    }
    
    class func initDatabase() {
        let key = "IsCreatedDb"
        let defaults = UserDefaults()
        if let value: Bool = defaults.value(forKey: key) as? Bool, !value{
            createUrlTable()
            defaults.setValue(true, forKey: key)
        }
    }
    
    class func createUrlTable() {
        let sql = "CREATE TABLE Address (address text)"
        KSYSQLite.sharedInstance.executeNonQuery(sql: sql)
    }
}
